opendatahub-io/data-science-pipelines-operator
# Upgrade argo images

## The issue resolved by this Pull Request:
 

- Updated Argo images with the newly built versions incorporating CVE fixes included in this PR- https://github.com/opendatahub-io/argo-workflows/pull/18
- Bumped kfp version to 2.10.1 since 2.10.0 is not supported with new argo images.

## Description of your changes:



## Testing instructions

-   Deploy DSPO
-  Deploy dspa, create a pipeline run and verify its successful completion.

## Checklist
